tests/src/Functional/DvfFieldBasicConfigTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Drupal\Tests\dvf\Functional;
+
+/**
+ * Class DvfFieldBasicConfigTest
+ *
+ * Functional tests for basic configuration of dvf fields.
+ * Validates that a dvf field can be configured with minimum settings,
+ * without errors and configured chart renders on the frontend.
+ *
+ * For more advanced configuration create a more specific test file.
+ *
+ * All tests use the default visualisation style
+ * set on parent class DvfFieldTestBase.
+ *
+ * @package Drupal\Tests\dvf\Functional
+ */
+class DvfFieldBasicConfigTest extends DvfFieldTestBase
```

Comment:
@jez500 I will add to this and refactor in a future PR to cover more configuration